ProcessLib/Process.h

```diff
@@ -162,7 +232,10 @@ class Process
 	}
 
 protected:
+	unsigned const _integration_order = 2;
```

Comment:
As it is, yes, in the initialization list.
Otherwise needs to be changed to non-constant...